changelog/changelog.go

```diff
@@ -1,12 +1,22 @@
 package changelog
 
 import (
+	"fmt"
 	"text/template"
 )
 
 const (
-	tplStr = `### {{.OldRelease}} -> {{.NewRelease}}
+	tplStr = `{{ if and .OldRelease .NewRelease }}
+### {{.OldRelease}} -> {{.NewRelease}}
+{{end}}
+
+{{ if (len .Migrations) gt 0 -}}
+### Migrations
```

Comment:
@mboersma yea, perhaps not the ideal term.  Either of the alternatives you mentioned sound good -- and also noticing that the header size should probably be bumped down to that of `Fixes` below.